build/default.conf

```diff
@@ -14,6 +14,11 @@ server {
         try_files $uri /assets/$1 =404;
     }
 
+    # Service Worker requests should not be changed
+    location ~* ^/ngsw {
+        try_files $uri =404;
+    }
```

Comment:
is that a manual request by user to ngsw/state or some internal ones by services?

Reply:
Its the ones for the `ngsw-worker.js` (so the actual service worker which we request in the `app.module.ts`) and the request for the `ngsw.json` which tracks the cached files and indicates updates which the service worker itself requests.